### PR TITLE
root file graphql

### PR DESCRIPTION
**Pull request by krishna1212031**

### CODE REVIEW

File: apollo-client.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { ApolloClient, InMemoryCache } from "@apollo/client";
+
+const client = new ApolloClient({
+    uri: "https://countries.trevorblades.com",
+    cache: new InMemoryCache(),
+});
+
+export default client;
```
Comment (SakshiShreya) [inline apollo-client.js:8]: Add new line at the end

File: apollo-client.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { ApolloClient, InMemoryCache } from "@apollo/client";
+
+const client = new ApolloClient({
+    uri: "https://countries.trevorblades.com",
```
Comment (SakshiShreya) [inline apollo-client.js:4]: Which url is this???

**Review by SakshiShreya — CHANGES_REQUESTED** [review]

File: apollo-client.js (hunk 2 above)
Comment (krishna1212031) [reply apollo-client.js:4]: done

**Review by SakshiShreya — APPROVED** [review]

File: apollo-client.js (hunk 1 above)
Comment (krishna1212031) [reply apollo-client.js]: added

**Pull request merged**